external-builds/pytorch/pytorch_torch_repo.py

```diff
@@ -43,7 +43,10 @@
 
 THIS_MAIN_REPO_NAME = "pytorch"
 THIS_DIR = Path(__file__).resolve().parent
-THIS_PATCHES_DIR = THIS_DIR / "patches" / THIS_MAIN_REPO_NAME
+
+DEFAULT_ORIGIN = "https://github.com/pytorch/pytorch.git"
+DEFAULT_HASHTAG = "main"
```

Comment:
```suggestion
DEFAULT_REPO_HASHTAG = "main"
```
i liked it how it was named below in `add_common()`